wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/WTabSetWebElementTest.java

```diff
@@ -27,7 +26,7 @@ public WTabSetWebElementTest() {
 	public void testTabFeatures() {
 		SeleniumWComponentsWebDriver driver = getDriver();
 
-		SeleniumWTabSetWebElement wTabSet = driver.findWTabSet(new ByTabSet());
+		SeleniumWTabSetWebElement wTabSet = driver.findWTabSet(new ByWTabSet());
```

Comment:
Whilst one may assume that a well formed UI should not include more than one tabset it is, unfortunately, unlikely that this is the case. I am not sure of the efficacy of the ByWTabSet extension of By (as noted above) but it could be useful if it was possible to differentiate different tabsets by some other mechanism. I am content that for the purposes of this test getting the first tabset is sufficient, and that if one wanted the nth tabset one could use `findWTabSets` and get the nth item but surely `findWTabSet` itself could be overloaded to accept an int argument to get the nth tabset **and** since the methods are `findWTabSet[s]` then the `By` argument is a bit superfluous? The only other sensible option is getting one with a specific ID.